cob_controller_configuration_gazebo/launch/robots/default_controllers_cob3-2.launch

```diff
@@ -117,67 +117,67 @@
 
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-1)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="laser_front"/>
-		<param name="topic_name" value="/scan_front"/>
+		<param name="topic_name" value="laser_front/scan_filtered"/>
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-2)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="laser_rear"/>
-		<param name="topic_name" value="/scan_rear"/>
+		<param name="topic_name" value="laser_rear/scan_filtered"/>
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-3)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="laser_top"/>
-		<param name="topic_name" value="/scan_top"/>
+		<param name="topic_name" value="laser_top/scan_filtered"/>
 	</node>
 	<node pkg="cob_controller_configuration_gazebo" type="fake_diagnostics.py" name="$(anon fake_diagnostics-4)" cwd="node" respawn="false" output="screen" >
 		<param name="diagnostics_name" value="/stereo/left/color_camera"/>
-		<param name="topic_name" value="/stereo/left/image_raw"/>
+		<param name="topic_name" value="stereo/left/image_raw"/>
```

Comment:
I disagree. In my opinion, for stereo cameras, the naming should be something right `/stereo_top_camera/left/image_raw` and `/stereo_top_camera/right/image_raw`.

The first part specifies which camera it is, similar to the laser scanners, whereas the following `left/right` tags actually specify which of the two mono cameras the data comes from. Changing this would (probably) break a lot of the algorithms for stereo vision that assume this setup...